### PR TITLE
nrt_observation table column name change and added new columns.

### PR DESCRIPTION
**Pull request by ssathiah**
## Notes

Please include a summary of the change and which issue is fixed or which feature is added. 

## JIRA

- **Related story**: [Jira Ticket](url)

## Checklist

- [ ] PR focuses on a single story.
- [ ] New unit tests added and ensured they pass.
- [ ] Service has been tested in local and it works as expected.
- [ ] Documentation has been updated for this code change (if needed).
- [ ] Code follows the Java Coding Conventions (https://www.oracle.com/java/technologies/javase/codeconventions-programmingpractices.html).

## Types of changes

What types of changes does this PR introduces?

- [ ] Bugfix
- [ ] New feature
- [ ] Breaking change

## Testing

- [ ] Does this PR has >90% code coverage?
- [ ] Is the screenshot attached for code coverage?
- [ ] Does the `gradle build` pass in your local? 
- [ ] Is the `gradle build` logs attached?

### CODE REVIEW

File: nnd-data-poll-service/src/main/java/gov/cdc/nnddatapollservice/rdbmodern/dto/NrtObservationDto.java [hunk 1]
```diff
@@ -220,4 +220,22 @@ public class NrtObservationDto {
 
     @SerializedName("max_datetime")
     private String maxDatetime;
```
Comment (upasanapattnaik-eq) [inline nnd-data-poll-service/src/main/java/gov/cdc/nnddatapollservice/rdbmodern/dto/NrtObservationDto.java:222]: maxDatetime is system column. Are there any issues when this is polled?

Comment (ssathiah) [reply nnd-data-poll-service/src/main/java/gov/cdc/nnddatapollservice/rdbmodern/dto/NrtObservationDto.java:222]: It should be ok and won't cause any problem as it's only in the dto class , not in jpa class. It's commented in the JPA class. I guess this field is not being used, we can cleanup later once DUC is available next week.

File: nnd-data-poll-service/src/main/resources/sql/rdb_modern/create_rdb_modern_table.sql [hunk 2]
```diff
@@ -51,10 +51,10 @@ CREATE TABLE dbo.nrt_observation (
 	txt varchar(1000) NULL,
 	interpretation_cd varchar(20) NULL,
 	interpretation_desc_txt varchar(100) NULL,
-	report_observation_id bigint NULL,
-	followup_observation_id nvarchar(MAX) NULL,
-	report_refr_id bigint NULL,
-	report_sprt_id bigint NULL,
+    report_observation_uid bigint NULL,
```
Comment (upasanapattnaik-eq) [inline nnd-data-poll-service/src/main/resources/sql/rdb_modern/create_rdb_modern_table.sql:54]: Nitpick on the alignment

File: nnd-data-poll-service/src/main/java/gov/cdc/nnddatapollservice/repository/rdb_modern/model/NrtObservation.java [hunk 3]
```diff
@@ -300,9 +318,14 @@ public NrtObservation(NrtObservationDto dto) throws ParseException {
         this.healthCareId = dto.getHealthCareId(); // Added this field
         this.morbHospReporterId = dto.getMorbHospReporterId(); // Added this field
         this.priorityCd = dto.getPriorityCd(); // Added this field
+        this.transcriptionistAuthType= dto.getTranscriptionistAuthType();
+        this.transcriptionistIdAssignAuth= dto.getTranscriptionistIdAssignAuth();
+        this.assistantInterpreterAuthType= dto.getAssistantInterpreterAuthType();
+        this.assistantInterpreterIdAssignAuth= dto.getAssistantInterpreterIdAssignAuth();
+        this.accessionNumber= dto.getAccessionNumber();
+        this.morbHospId= dto.getMorbHospId();
```
Comment (sveselev) [inline nnd-data-poll-service/src/main/java/gov/cdc/nnddatapollservice/repository/rdb_modern/model/NrtObservation.java:326]: Looks like NrtObservation  still does not take all the NrtObservationDto fields?
What about `reportObservationUid`, `followupObservationUid`, `reportRefrUid`, `reportSprtUid`, `morbPhysicianId`, `morbReporterId`.

File: nnd-data-poll-service/src/main/resources/sql/rdb_modern/create_rdb_modern_table.sql [hunk 4]
```diff
@@ -1,79 +1,86 @@
...
+    refresh_datetime                     datetime2(7) GENERATED ALWAYS AS ROW START      NOT NULL,
+    max_datetime                         datetime2(7) GENERATED ALWAYS AS ROW END HIDDEN NOT NULL,
+    transcriptionist_auth_type           varchar(100) NULL,
+    transcriptionist_id_assign_auth      varchar(199) NULL,
+    assistant_interpreter_auth_type      varchar(100) NULL,
+    assistant_interpreter_id_assign_auth varchar(199) NULL,
+    accession_number                     varchar(199) NULL,
+    morb_hosp_id                         bigint NULL,
     PERIOD FOR SYSTEM_TIME (refresh_datetime, max_datetime)
 );
```
Comment (sveselev) [inline nnd-data-poll-service/src/main/resources/sql/rdb_modern/create_rdb_modern_table.sql:86]: 👍🏻

File: nnd-data-poll-service/src/main/resources/sql/rdb_modern/create_rdb_modern_table.sql (hunk 2 above)
Comment (ssathiah) [reply nnd-data-poll-service/src/main/resources/sql/rdb_modern/create_rdb_modern_table.sql]: Fomatted

**Review by sveselev — APPROVED** [review]

**Pull request merged**